src/components/LocationSearch/LocationSearch.jsx

```diff
@@ -0,0 +1,83 @@
+import { useState } from "react";
+import PropTypes from "prop-types";
+import Close from "../../assets/icons/close.png";
+import Search from "../../assets/icons/search.png";
+import LocationItem from "../Locationitem/LocationItem";
+import ErrorMessage from "../ErrorMessage/ErrorMessage";
+import "./LocationSearch.css";
+
+const LocationSearch = ({
+  handleLocationSearchVisibility,
+  error,
+  isLocationSearchOpen,
+  locations,
+  handleListSearch,
+  handleLocationSearch,
```

Comment:
Also, I like to put the handles always in the end, like:

```suggestion
  error,
  locations,
  isLocationSearchOpen,
  handleLocationSearch,
  handleLocationSearchVisibility
```